src/spritekit.cs

```diff
@@ -285,9 +305,11 @@ partial interface SKNodeEvent_NSEvent {
 	[Category, BaseType (typeof (UITouch))]
 	partial interface SKNodeTouches_UITouch {
 
+		[NoWatch]
```

Comment:
Since the category would be empty on watchOS let's move the `[NoWatch]` on the type. If that change in the future then it's easy to fix (and won't cause a breaking change).